torch/cuda/graphs.py

```diff
@@ -46,12 +46,38 @@ def graph_pool_handle():
 class CUDAGraph(torch._C._CUDAGraph):
     r"""Wrapper around a CUDA graph.
 
+    Arguments:
+        instantiate_eagerly (bool, optional): if True, a
+        cudaGraphExec_t will be created at the end of capture_end(),
+        which is the default behavior.
+
+            Previously, the graph would always and immediately be
+            instantiated at the end of capture_end(). Then, the
+            cudaGraph_t would be destroyed. However, that prevents
+            users from modifying the cudaGraph_t before the
+            cudaGraphExec_t is made. We now always retain the
+            cudaGraph_t so that it can be modified. If a user does
+            modify a cudaGraph_t, we would like to allow them to opt
+            out of having to instantiate twice. If
+            instantiate_eagerly=True, on the first call to replay(),
+            instantiate() will be called if it hasn't been called
+            already. Since replay() is expected to be run on
+            performance-critical paths, we want to retain the prior
+            behavior of instantiating at the end of graph capture to
+            prevent a latency spike on the first call of replay() in
+            existing latency-sensitive code in the common case where
+            the user is not modifying the cudaGraph_t. If a user knows
+            that they will be modifying the cudaGraph_t after graph
+            capture is done, they should set instantiate_eagerly to
+            true to avoid the overhead of instantiating twice. They
+            can call instantiate() manually before replay() to prevent
+
     .. warning::
         This API is in beta and may change in future releases.
     """
 
-    def __new__(cls):
-        return super().__new__(cls)
+    def __new__(cls, instantiate_eagerly=True):
```

Comment:
https://github.com/pytorch/pytorch/blob/cadcb5d36802579441da3dbe3c2eecc74a0eae78/torch/_C/__init__.pyi.in#L2280 is probably what mypy is complaining about

Reply:
Ah, thanks. I had modified __init__.pyi locally and was horribly confused why mypy was showing these errors only in CI.